demo/chbench/mzcompose.yml

```diff
@@ -352,12 +354,17 @@ mzconduct:
         duration: 64800
       - step: stop-services
         services: [chbench, postgres]
-      # Give Materialize a few seconds to catch up with topics / drain
+      # Give Materialize a few seconds to catch up with topics / drain before snapshotting views
       - step: sleep
         duration: 15
```

Comment:
This step was added in #4885. Ideally we could remove the sleep calls because they are kludges. Unfortunately, they appear to be load-bearing kludges to get this stuff off the ground.